web_export_view/static/src/xml/web_export_view_template.xml

```diff
@@ -1,11 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
-
 <templates id="template" xml:space="preserve">
-<t t-name="WebExportTreeViewXls">
-    <div class="o_cp_sidebar btn-group btn-group-sm">
-        <button class="btn btn-default export_treeview_xls" type="button" title="Export xls">
-            <i t-translation="off" class="fa fa-file-excel-o"></i>
-        </button>
-    </div>
-</t>
+	<t t-name="WebExportTreeViewXls">
+    	<div class="o_cp_sidebar btn-group btn-group-sm">
+        	<button class="btn btn-default export_treeview_xls" type="button" title="Export xls">
```

Comment:
Please do not add tab characters, only spaces please.

Reply:
@valtterilattu possible resolve this ? or @astirpe can you be permissive for tabs ?